### DISCUSSION TITLE
No FX Device Detected Even after reinstalling OS

### DISCUSSION THREAD

**Discussion by ZvSmithvZ** [T-Troll/alienfx-tools#287] [Q&A]
I downloaded and ran your AlienFx-tools package one time. I experimented with it and decided it didn't really have the compatibility with my Aurora R11 that I was looking for. Ever since I ran your program, I am unable to get Alienware Command Center to detect that my system has lights. I wiped my C drive and reinstalled the AW stock firmware. That didn't solve my issue. I wiped again and reinstalled basic windows 10. That also still didn't solve the issue. Do you have any ideas on how to restore my pc to a working state where I am able to use AWCC again? I had no idea that AlienFX-tools was going to wipe my system's ability to be recognized as an AlienFX-capable device.

**Comment by T-Troll** [comment]
> I experimented with it and decided it didn't really have the compatibility with my Aurora R11

How do you configure it out? It should have, in fact.

> That didn't solve my issue. 

The only thing can get broken is chip hang (Latest AWCC have a bug about it). As mentioned in [readme](https://github.com/T-Troll/alienfx-tools#readme), you can try to reset light system:
- Shutdown it
- Remove ALL cables, including power cord
- (Because you have desktop, not needed for notebooks) Push power button for 15 seconds
- Wait another 15 seconds
- Connect cables, start system back

You will see rainbow lights at start in case reset is successful.

But better roll back AWCC to early version (October-November'22) first.

**Reply by ZvSmithvZ** [reply]
No rainbow lights. I have done that several times.

**Reply by ZvSmithvZ** [reply]
I reinstalled the AlienFX Control. It only shows fans and power tab, profiles, and settings. No lights tab at all.

**Reply by T-Troll** [reply]
This means you don't have any light device into your system.
Check Device Manager about broken/non-working device - look for USB HID device ("vendor-defined") with VID_187c&PID_0550.

PS: Some years ago, 17R4 (not m17, previous model) have the issue - AWCC can break light chip firmware at update, and no way to recover it. Sometimes they return...
